app/display/runtime/src/main/java/org/csstudio/display/builder/runtime/script/internal/RuntimeScriptHandler.java

```diff
@@ -45,7 +45,8 @@
     private final Widget widget;
     private final List<ScriptPV> infos;
     private final Script script;
-    private final boolean check_connections;
+    private final boolean is_rule;
+    private boolean check_connections;

```

Comment:
`check_connections` was `final`. Now it's changed at runtime, on different threads, so needs to be `volatile`.

Reply:
Oops, you are right...